.github/workflows/test.yaml

```diff
@@ -1,24 +1,26 @@
 name: Test
 on:
-  workflow_run:
-    workflows: [Format]
-    types:
-      - completed
+  push:
+    branches: [ $default-branch ]
```

Comment:
Same comments on when to run this as in another PR. Let's please run this for all branches.

Reply:
+1, forgot to reflect this change here.